pkg/restapi/comparator/operation/compare.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package operation
+
+import (
+	"net/http"
+
+	"github.com/trustbloc/edge-service/pkg/restapi/comparator/operation/openapi"
+)
+
+// HandleEqOp handles a ComparisonRequest using the EqOp operator.
+func (o *Operation) HandleEqOp(w http.ResponseWriter, op *openapi.EqOp) {
+	comparison := &openapi.Comparison{Result: true}
+
+	for i := range op.Args() {
+		query := op.Args()[i]
+
+		switch q := query.(type) {
+		case *openapi.DocQuery:
+			docMeta, err := o.vaultClient.GetDocMetaData(*q.DocID, *q.VaultID)
+			if err != nil {
+				respondErrorf(w, http.StatusInternalServerError, "failed to get doc meta: %s", err.Error())
+
+				return
+			}
+
+			logger.Infof(docMeta.URI)
+
+			// TODO where to put docMeta URI in doc query struct for doc1
```

Comment:
Do you mean [this](https://github.com/trustbloc/edge-service/blob/8443964417f1e806286514030bb40edf28456e2a/pkg/restapi/csh/operation/openapi/doc_query.go#L22-L39)?

Reply:
Yes